docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -W
```

Comment:
This flag makes warnings be treated as errors. Should have put this info int he commit msg